userspace/libscap/scap_suppress.h

```diff
@@ -33,6 +33,13 @@ typedef struct scap_tid
 	UT_hash_handle hh; ///< makes this structure hashable
 } scap_tid;
 
+// Famous last words: will never have more than 1024 CPUs... :-)
+#define SCAP_CPUID_MAX (1024)
+
+typedef struct scap_key_value_cache {
+	uint64_t key;
```

Comment:
Instead of using generic names key/val, could we use a name like `tid` for the key and `suppressed_tid` for the value? (I think that's what the value is? Trying to remember this code).

I know you gave the type a generic name so it can be used later, but I'd lean towards a more specific names as this is only used for suppressed tids at the moment.

Reply:
Thanks for the review and the suggestion! Pushed the change.